DataTypes/Program.cs

```diff
@@ -1,12 +1,145 @@
+                        Console.WriteLine("| IF THE SPIDER HITS HIGHER THAN YOU, YOU WILL DIE |");
+                        Console.WriteLine("|==================================================|");
+                        Thread.Sleep(2000);
+                        Random randm = new Random();
+                        int pDmg = randm.Next(3,10);
+                        int gsDmg = randm.Next(1,5);
+                        Console.WriteLine("You hit a, {0}", pDmg);
+                        Console.WriteLine("Giant Spider hit a, {0}", gsDmg);
+                        Thread.Sleep(2000);
+
+                        if (gsDmg > pDmg)
+                        {
+                            Console.WriteLine("The giant spider had dealt more damage than you!");
+                            bool live = false;
+                            if (live == true)
+                            {
+                                Console.WriteLine("You escape alive!");
+                            }
+                            else
+                            {
+                                Console.WriteLine("You escape dead, you know soul leaving the body and all that rubbish");
+                            }
+                        }
+                        else if (pDmg < 5)
+                        {
+                            Console.WriteLine("You didn't do enough damage to kill the spider, but you manage to escape.");
+                            bool live = false;
+                            if (live == true)
+                            {
+                                Console.WriteLine("You escape alive!");
+                            }
+                            else
+                            {
+                                Console.WriteLine("You escape dead, you know soul leaving the body and all that rubbish");
+                            }
+                        }
+                        else
+                        {
+                            Console.WriteLine("You killed the giant spider!");
+                            bool live = true;
+                            if (live == true)
+                            {
+                                Console.WriteLine("You escape alive!");
+                            }
+                            else
+                            {
+                                Console.WriteLine("You escape dead, you know soul leaving the body and all that rubbish");
+                            }
+                        }
+                    }
+                }
+                // Don't Fight Spider
+                else
+                {
+                    Console.WriteLine("You choose not to fight the giant spider.");
+                    Thread.Sleep(2000);
+                    Console.WriteLine("As you turn away, it ambushes you and impales you with it's fangs!");
+                    bool live = false;
+
```

Comment:
Since you're setting live = false before the conditional statement, the conditional is unnecessary. In this instance live will never == true, and the subsequent code won't be run.